appinfo/Migrations/Version20180604132522.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace OCA\notifications\Migrations;
```

Comment:
shouldn't "notifications" be "Notifications" ?

Reply:
The rest of the migrations have the same "problem". Either leave it or fix it everywhere. I wonder if there are more apps with the same issue...

Reply:
only cosmetic - the classes are never loaded/referenced by the namespace